src/test/java/com/bulkbuy/enterprise/EnterpriseApplicationTests.java

```diff
@@ -58,7 +58,7 @@ void verifyOrderProperties() {
      * Confirm saving orders and pulling a full list of orders.
      */
     @Test
-    void confirmSavedOrder(){
+    void confirmSavedOrder() {
```

Comment:
need a space after “()” and before “{“
https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3